### PR TITLE
add method to manage alias

### PR DESCRIPTION
**Pull request by phiz71**
**Issue**

https://github.com/gravitee-io/issues/issues/7110

**Description**

add method to manage alias

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `3.8.5-7110-fix-ILM-support-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/elasticsearch/gravitee-common-elasticsearch/3.8.5-7110-fix-ILM-support-SNAPSHOT/gravitee-common-elasticsearch-3.8.5-7110-fix-ILM-support-SNAPSHOT.zip)

### CODE REVIEW

File: src/main/java/io/gravitee/elasticsearch/client/Client.java [hunk 1]
```diff
@@ -38,4 +40,7 @@ public interface Client {
     Completable putPipeline(String templateName, String template);
     Single<SearchResponse> search(String indexes, String type, String query);
     Single<CountResponse> count(String indexes, String type, String query);
+
+    Single<JsonNode> getAlias(String aliasName);
```
Comment (brasseld) [inline src/main/java/io/gravitee/elasticsearch/client/Client.java:44]: Should return a Maybe to differentiate technical issue from no alias found.

**Review by brasseld — CHANGES_REQUESTED** [review]

File: src/main/java/io/gravitee/elasticsearch/client/http/HttpClient.java [hunk 2]
```diff
@@ -320,6 +323,60 @@ public Completable putTemplate(String templateName, String template) {
...
+                    response.bodyAsString() +
+                    "]"
+                );
+            });
+    }
+
+    @Override
+    public Completable createIndexAndAlias(String indexName, String aliasName) {
+        String template =
+            "{\n" +
```
Comment (brasseld) [inline src/main/java/io/gravitee/elasticsearch/client/http/HttpClient.java:351]: That hurts my 👀

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**